submission/persistence-mongo/src/test/kotlin/ac/uk/ebi/biostd/persistence/doc/db/data/SubmissionDocDataRepositoryTest.kt

```diff
@@ -55,13 +63,32 @@ internal class SubmissionDocDataRepositoryTest {
     inner class ExpireSubmissions {
         @Test
         fun `expire active processed versions`() {
+            val referencedFile = tempFolder.createFile("referenced.txt")
+            val file = createNfsFile("referenced.txt", "Files/referenced.txt", referencedFile)
+            val fileListFile = FileListDocFile(
+                id = ObjectId(),
+                submissionId = testDocSubmission.id,
+                file = file.toDocFile(),
+                fileListName = "test-file-list",
+                index = 1,
+                submissionVersion = 1,
+                submissionAccNo = "S-BSST4"
+            )
+
             testInstance.save(testDocSubmission.copy(accNo = "S-BSST4", version = -1))
+            fileListDocFileRepo.save(fileListFile)
+
             testInstance.save(testDocSubmission.copy(accNo = "S-BSST4", version = 2))
+            fileListDocFileRepo.save(fileListFile.copy(submissionVersion = 2))
 
-            testInstance.expireActiveProcessedVersions("S-BSST4")
+            testInstance.expireVersions(listOf("S-BSST4"))
 
             assertThat(testInstance.getByAccNoAndVersion("S-BSST4", version = -1)).isNotNull
             assertThat(testInstance.getByAccNoAndVersion("S-BSST4", version = -2)).isNotNull
+            assertThat(
```

Comment:
I think we should test file now is negative rather than non positive version file exists. Assertions looks similar but the second one is more direct.